package.json

```diff
@@ -115,6 +115,7 @@
     "@fortawesome/fontawesome-svg-core": "~1.2.25",
     "@fortawesome/free-solid-svg-icons": "~5.11.2",
     "@fortawesome/react-fontawesome": "~0.1.4",
+    "@hospitalrun/components": "~0.4.3",
```

Comment:
Actually we don't need this dependency. It was a tsconfig.json issue. It is fixed on master, now.